pages/ksphere/kommander/1.1.0-beta/operations/access-control/index.md

```diff
@@ -9,29 +9,31 @@ excerpt: Centrally manage access across clusters
 
 Kommander has two conceptual groups of resources that are used to manage access control:
+
 - Kommander roles: control access to resources on the management cluster.
 - Cluster Roles, Workspace Roles, and Project Roles: control access to resources on all target clusters in scope.
 
 These two groups of resources can be used to manage access control within 3 levels of scope:
 
 |           | Kommander Roles                                                                                         | Global/Workspace/Project Roles                                                                           |
-|-----------|---------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| --------- | ------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
 | Global    | Global Kommander Roles create ClusterRoles on the management cluster.                                   | Global Cluster Roles federates ClusterRoles on all target clusters across all workspaces.                |
-| Workspace | Workspace Kommander Roles create namespaced Roles on the management cluster in the workspace namespace and workspace project namespaces. | Workspace Cluster Roles federates ClusterRoles on all target clusters in the workspace.                  |
+| Workspace | Workspace Kommander Roles create namespaced Roles on the management cluster in the workspace namespace. | Workspace Cluster Roles federates ClusterRoles on all target clusters in the workspace.                  |
 | Project   | Project Kommander Roles create namespaced Roles on the management cluster in the project namespace.     | Project Roles federates namespaced Roles on all target clusters in the project in the project namespace. |
 
 The policies for each level and type create RoleBindings or ClusterRoleBindings on the clusters that apply to each category.
 
 This approach gives you maximum flexibility over who has access to what resources, conveniently mapped to your existing identity providers' claims.
 
+<p class="message--note"><strong>NOTE: </strong>In addition to granting a Kommander Role, you must also grant the appropriate opsportal role to allow external users and groups into the UI. See <a href="/ksphere/konvoy/1.5.0-beta/security/external-idps/rbac/#portal-authorization">Konvoy RBAC Documentation</a> for details about the built-in opsportal roles. This role may be automatically added to Kommander role binding subjects in future versions of Kommander.</p>
```

Comment:
is this how we handle links? is there some way to get the version out of here?

Reply:
Yeah, unfortunately we have to come back and change them all close to release. An alternative way would have been to maintain a /latest/ directory and duplicate/rename it whenever there is a new release. But as it is we need to go back and correct a bunch of these in both konvoy and kommander

Reply:
This is because we don't have 1.5 beta set to latest until GA. We can rethink that @jaxesn because then these links can be set to latest and won't 404.